modules/search/instagram.py

```diff
@@ -19,76 +19,138 @@
 
 meta = {
 	'name': 'Instagram Search',
-	'author': 'Aman Singh',
-	'version': '0.2',
+	'author': 'Rishabh Jain, Aman Singh',
```

Comment:
May I ask the collaborator and the maintainer to not allow this? 

If i may defend @prince-7 's efforts on authoring this module, another fellow bug fixer can't have his name up with his! 
If so, then all of us should rewrite modules and rename the original authors! It's not how it works, right?
The main author should always be credited for his thought and valuable addition, not any other consequent bug fixer

Reply:
depends on @prince-7?

Reply:
The only issue I have is about uniformity because in the earlier modules only the primary author's name is mentioned. Maybe we can have a 'contributors' section somewhere in the project/readme where all the contributors are named so that their efforts are recognized.

Reply:
good. rishabhjainfinal will add the contributor section.